pcdsdevices/pseudopos.py

```diff
@@ -167,14 +167,14 @@ class Parallel(SyncAxesBase):
     """
 
     pseudo = Cpt(PseudoSingleInterface)
+    _offsets = None
```

Comment:
What are your thoughts on this later becoming a notepad signal in some form?

Reply:
It's plausible to make this a notepad signal, but it's even better to statically define it for the class

Reply:
Right, it should be defined per class, but for classes where they could be variable, it might be good to make their offsets recorded as PVs, right?

Reply:
I'm not sure how to do this generically for a parent class that has no idea how many real axes there will be in the subclass

Reply:
Sorry I'm not being very clear. I'm not requesting this be added here. I'm just thinking about whether it would be beneficial for the child classes to implement something like that in the future.

Reply:
This would be useful in the cases where your desired offsets change during an experiment